src/Struct/ContextAdyenDataStruct.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <shopware@adyen.com>
+ */
+
+namespace Adyen\Shopware\Struct;
+
+use Shopware\Core\Framework\Struct\Struct;
+
+class ContextAdyenDataStruct extends Struct
```

Comment:
Lets align the names of these struct extensions, like in #162 , let's call this `AdyenContextDataStruct`

Reply:
@peterojo done. I thought I used the same name scheme but apparently not :smile: